t/mojo/loader.t

```diff
@@ -72,6 +72,31 @@ subtest 'Search packages' => sub {
   is_deeply [find_packages 'MyLoaderTest::DoesNotExist'], [], 'no packages found';
 };
 
+# is_package
+# To check cases use
+# perl -e"package aaa::a; print __PACKAGE__"
```

Comment:
You forgot to remove this.

Reply:
I wanted to leave it for those who need to add tests in the future

Reply:
We don't leave comments like that around.